operator/src/main/resources/Operator.properties

```diff
@@ -144,6 +144,12 @@ WLSKO-0191=Rolling restart of domain {0} completed
 WLSKO-0192=Executing make right domain operation, recheck count for server {0} is {1}.
 WLSKO-0193=Waiting for server {0} to start, recheck count is {1}.
 WLSKO-0194=Internal identity initialization step failed with exception {0}.
+WLSKO-0195=Introspection encountered a fatal error and will not retry automatically. \
+  Please resolve the error and then update 'domain.spec.introspectVersion' to force a retry.
+WLSKO-0196=Stop introspection retry - exceeded configured domainPresenceFailureRetryMaxCount: {0}. The \
+  domainPresenceFailureRetryMaxCount is an operator tuning parameter and can be controlled by adding it to the \
```

Comment:
Can domainPresenceFailureRetryMaxCount also be controlled via helm upgrade command?

Reply:
AFAIK, this can't be done using `helm upgrade` command.